{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -11,18 +11,35 @@ replace = version='{new_version}'
 search = __version__ = '{current_version}'
 replace = __version__ = '{new_version}'
 
-[bdist_wheel]
-universal = 1
```

Comment:
all changes in this `setup.cfg` is probably fairly specific to us, not applicable to upstream.

Reply:
The `bdist_wheel` flag is no longer relevant. It's only meant for packaging python2/3-compatible packages. https://packaging.python.org/guides/distributing-packages-using-setuptools/#universal-wheels

Reply:
Yes sorry unclear my bad, this universal wheel is okay, I meant the rest of the file, like all the specific flakes8 ignore rules, that's more specific to us.  I should have chosen a different diff hunk to put my comment.  There is no file wide comment so I had to attach to one of the diff hunk.

Reply:
Ah, right. The flake8 overrides are needed specifically for black and flake8 to play nicely. If we were to push black into upstream, _most_ of these changes would be needed. Worth considering.

Reply:
Oh did not realize these flake8 ignore are for black.  Then yes if we push black upstream it should go together.  Same as for all the changes in tox.ini, I think most is for black.

So we should have a few separate PR for upstream, one for the typo, deprecation stuff, very high chance of acceptance, 1 for the Windows and Mac Travis-CI support, good chance of acceptance, and finally all things related to black, probably make it a toggle so higher acceptance chance.

In general small, easy to wrap our head around and focused PR have higher chance of acceptance than big, hard to follow which diff hunks are needed for which intend PR.